presto-oracle/src/test/java/io/prestosql/plugin/oracle/OracleQueryRunner.java

```diff
@@ -54,11 +54,12 @@ public static DistributedQueryRunner createOracleQueryRunner(
             queryRunner.createCatalog("tpch", "tpch");
 
             Map<String, String> connectorProperties = new HashMap<>();
-            connectorProperties.putIfAbsent("connection-url", server.getJdbcUrl() + (remarksReportingEnabled ? "?remarksReporting=true" : ""));
+            connectorProperties.putIfAbsent("connection-url", server.getJdbcUrl());
             connectorProperties.putIfAbsent("connection-user", TEST_USER);
             connectorProperties.putIfAbsent("connection-password", TEST_PASS);
             connectorProperties.putIfAbsent("allow-drop-table", "true");
             connectorProperties.putIfAbsent("oracle.connection-pool.enabled", String.valueOf(connectionPoolEnabled));
+            connectorProperties.putIfAbsent("oracle.remarks-reporting.enabled", String.valueOf(remarksReportingEnabled));
```

Comment:
i'd suggest removing `remarksReportingEnabled` parameter from `createOracleQueryRunner` and using connectorProperties for that.

can be a follow up (=> issue?)

Reply:
Do you mean?
`boolean connectionPoolEnabled,  boolean remarksReportingEnabled)` => `Map<String, Object> extraConnectorProperties`

(I assume if we get rid of `remarksReportingEnabled` we should also get rid of `connectionPoolEnabled`)

Reply:
// TODO(https://github.com/prestosql/presto/issues/5721) use extra